myosuite/envs/myo/assets/leg/myoosl_runtrack_p2.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" ?>
+<mujoco model="MyoSuite's MyoTrack">
+<!-- =================================================
+    Copyright 2024 MyoSuite
+    Model   :: MyoTrack (MuJoCoV3.1.2)
+    Author  :: Vikash Kumar (vikashplus@gmail.com), Vittorio Caggiano, Pierre Schumacher, Chun Kwang Tan
+    source  :: https://github.com/MyoHub/MyoSuite
+    License :: Under Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+====================================================== -©vk©-->
+
+    <include file="../../../../simhive/myo_sim/torso/assets/myotorsorigid_assets.xml"/>
+    <include file="../../../../simhive/myo_sim/osl/assets/myolegs_osl_assets.xml"/>
+    <include file="../myosuite_track_p2.xml"/>
+    <compiler meshdir='../../../../simhive/myo_sim/' texturedir='../../../../simhive/myo_sim/'/>
+
+    <asset>
+        <hfield name="terrain" size="1 35 1 0.001" nrow="600" ncol="20"/>
+        <texture name="texmyo" type="cube" file="scene/myosuite_icon.png"/>
+        <material name="matmyo" texture="texmyo" rgba="1 1 1 1"/>
+    </asset>
+
+    <worldbody>
+        <geom name="terrain" type="hfield" hfield="terrain" pos="0 0 -0.005" material="matfloor" conaffinity="1" contype="1" rgba=".9 .9 .9 1"/>
+
+        <camera name="side_view" pos="-3 0 2.25" xyaxes="0 -1 0 1 0 2" mode="trackcom"/>
+
+        <!-- ======= MyoAgent ======= -->
```

Comment:
not sure if we standardized on this but Agent might indicate more the controller. I would say MyoActor or MyoBody to indicate only the body/actuation side

Reply:
has been addressed